croud/logout.py

```diff
@@ -41,4 +43,15 @@ def logout(args: Namespace) -> None:
 
 async def make_request(env: str, token: str) -> None:
     async with HttpSession(env, token) as session:
-        await session.logout()
+        await session.logout(_logout_url(env))
+
+
+def _logout_url(env: str) -> str:
+    if env == "local":
+        domain = "http://localhost:8000"
+    else:
+        host = "cratedb.cloud"
```

Comment:
You could re-use CLOUD_PROD_DOMAIN from https://github.com/crate/croud/pull/96/files#diff-fc96a85f2127221b6aa3b3a312e082a9R30

```suggestion
        domain = env == "dev" and CLOUD_DEV_DOMAIN or CLOUD_PROD_DOMAIN
```